config/application.rb

```diff
@@ -22,5 +22,8 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    # Which is the lowest balance you should be ashamed of.
+    config.shameful_balance = 50 # In eurocents!
```

Comment:
I think `config.x.shameful_balance` is a more conventional way to put custom configs. See http://guides.rubyonrails.org/configuring.html#custom-configuration